save-demo/src/main/kotlin/com/saveourtool/save/demo/runners/Runner.kt

```diff
@@ -7,7 +7,7 @@ import java.nio.file.Path
 /**
  * Interface should be implemented by all the runners.
  */
-interface Runner<in P : DemoAdditionalParams, out R : DemoResult> {
+interface Runner<in P : DemoAdditionalParams, in K, out R : DemoResult> {
```

Comment:
Is `K` nullable? 

If not, probably `K : Any` then?

Reply:
It is not, will do as you suggested

Reply:
Done